docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
       ES_JAVA_OPTS: "-Xmx1G"
       discovery.type: "single-node"
   cloaked-search:
-    image: "gcr.io/ironcore-images/cloaked-search:latest"
+    image: "gcr.io/ironcore-images/cloaked-search:0.3.0"
```

Comment:
I think is rather leave this as latest so we don't have to update this with every release.